desktop/apps/categories3/components/FeaturedGenes.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import FeaturedGene from './FeaturedGene'
+
+const propTypes = {
+  featuredGenes: PropTypes.object
+}
+
+const Layout = styled.div`
+  padding-top: 1em;
+  column-count: 3;
+  column-gap: 2em;
+`
+
+const FeaturedGenes = ({ featuredGenes }) => {
+  return (
+    <Layout>
+      {featuredGenes
+        ? featuredGenes.genes.length > 0
+          ? featuredGenes.genes
+              .map(featuredGene => <FeaturedGene key={featuredGene.id} {...featuredGene} />)
+              .slice(0, 3)
+          : <p style={{ color: 'orange' }}>
+              missing Featured Links?<br />(No featuredGenes.genes list)
+            </p>
+        : <p style={{ color: 'red' }}>
+            missing Set?<br />(No featuredGenes object)
```

Comment:
All this "missing" stuff is to give us some helpful debug output while all the metadata for this feature gets entered into old admin.

<img width="1101" alt="missing" src="https://user-images.githubusercontent.com/140521/29851644-3015f6b8-8d03-11e7-85e6-3fcb4a7bd2ea.png">